ext/proj4_c_impl/main.c

```diff
@@ -224,13 +203,20 @@ static VALUE method_proj4_get_geographic(VALUE self)
   VALUE result;
   RGeo_Proj4Data *new_data;
   RGeo_Proj4Data *self_data;
+  PJ *geographic_proj;
 
   result = Qnil;
   new_data = ALLOC(RGeo_Proj4Data);
   if (new_data) {
     TypedData_Get_Struct(self, RGeo_Proj4Data, &rgeo_proj4_data_type, self_data);
 
-    new_data->pj = proj_crs_get_geodetic_crs(PJ_DEFAULT_CTX, self_data->pj);
+    geographic_proj = proj_crs_get_geodetic_crs(PJ_DEFAULT_CTX, self_data->pj);
+    if(geographic_proj == 0) {
```

Comment:
```suggestion
    if (geographic_proj == 0) {
```

The suggestion is not so important in itself, however, shouldn’t we consider introducing  a c styleguide to our projects at some point? In which case this could be an open issue to tackle post v3 (at least)